launchable/commands/record/session.py

```diff
@@ -149,3 +181,49 @@ def session(
             raise e
         else:
             click.echo(e, err=True)
+
+    if session_name:
+        try:
+            add_session_name(
+                client=client,
+                build_name=build_name,
+                session_id=session_id,
+                session_name=session_name,
+            )
+        except Exception as e:
+            if os.getenv(REPORT_ERROR_KEY):
+                raise e
+            else:
+                click.echo(e, err=True)
+
+
+def add_session_name(
+    client: LaunchableClient,
+    build_name: str,
+    session_id: str,
+    session_name: str,
+):
+    sub_path = "builds/{}/test_sessions/{}".format(build_name, session_id)
+    payload = {
+        "name": session_name
+    }
+    res = client.request("patch", sub_path, payload=payload)
+
+    if res.status_code == HTTPStatus.NOT_FOUND:
+        click.echo(
+            click.style(
+                "Test session {} was not found. Record session may be failed.".format(session_id),
```

Comment:
`Record session may have failed.` might be better, since the `record session` is done before this request.